### PR TITLE
Removing unused video_player dependency

### PR DESCRIPTION
**Pull request by wszeborowskimateusz**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -3,6 +3,4 @@
 | Platform | Example interface                                                      | Sample tweet                                                           |
 |:--------:|:--------------------------------------------------------------------|:------------------------------------------------------------------------|
 | Android  | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/android_main.png) | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/android_example.png) |
-|   iOS    | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/ios_main.png)    | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/ios_example.png)    |
-
-> Video/GIF Tweets don't work on iOS simulators ([video_player ios instructions](https://pub.dev/packages/video_player#ios))
```
Comment (kawa89) [inline example/README.md:8]: Is this sentence not true anymore:
> Video/GIF Tweets don't work on iOS simulators 

If it's still valid, we should leave it

File: README.md [hunk 2]
```diff
@@ -13,9 +13,6 @@ import it:
 
 `import 'package:tweet_ui/tweet_ui.dart';`
 
-If you want to show tweets with videos: check the
-[video_player installation](https://pub.dev/packages/video_player#installation) site.
```
Comment (kawa89) [inline README.md:17]: Maybe we could point to: https://jhomlala.github.io/betterplayer/#/install for instructions on better_player?

Comment (wszeborowskimateusz) [reply README.md:17]: Added

File: example/README.md (hunk 1 above)
Comment (wszeborowskimateusz) [reply example/README.md]: Reverted it

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- [BREAKING] - Change onTapImage to OnTapImage to remove analyser warning
+- Removing lock file from repository - adding it to gitignore
+- Updating ios version for example project to 11 to support better player
+- Regenerating example project with flutter create command
+- Removing unused video_player dependency
```
Comment (kawa89) [inline CHANGELOG.md:13]: I think we can divide the Changelog a bit more, to use `### Removed` for `Removing unused video_player dependency` and the lock file.

**Review by kawa89 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (wszeborowskimateusz) [reply CHANGELOG.md:13]: done

**Pull request merged**